supportedBackends/sfs/src/main/scala/cromwell/backend/impl/sfs/config/MemoryDeclarationValidation.scala

```diff
@@ -3,10 +3,10 @@ package cromwell.backend.impl.sfs.config
 import cromwell.backend.MemorySize
 import cromwell.backend.validation._
 import wdl.expression.NoFunctions
-import wdl4s.parser.MemoryUnit
-import wdl.types._
-import wdl.values.{WdlFloat, WdlInteger, WdlOptionalValue, WdlValue}
 import wdl.{Declaration, NoLookup, WdlExpression}
```

Comment:
NotYourProblem: I think these probably need to be refactored to use `WomExpression` at some point